packages/main/src/plugin/filesystem-monitoring.ts

```diff
@@ -41,11 +41,6 @@ export class FileSystemWatcherImpl implements containerDesktopAPI.FileSystemWatc
       this._onDidCreate.fire(uri);
     });
 
-    this.watcher.on('addDir', (addedPath: string) => {
```

Comment:
It is suppose to be included ?

Reply:
Last minute rebase failure - force pushed to fix.